lib/addons/istio-addons/istio-ingress-gateway.ts

```diff
@@ -0,0 +1,27 @@
+import { Construct } from 'constructs';
+import { ClusterInfo } from "../../spi";
+import { HelmAddOn, HelmAddOnProps } from "../helm-addon";
+import { dependable, supportsALL } from '../../utils';
+
+const defaultProps: HelmAddOnProps = {
+    name: 'istio-ingressgateway',
+    release: 'ingressgateway',
+    namespace: 'istio-system',
+    chart: 'gateway',
+    version: "1.20.1",
+    repository: 'https://istio-release.storage.googleapis.com/charts',
+    values: {},
+};
+
+@supportsALL
+export class IstioIngressGatewayAddon extends HelmAddOn {
```

Comment:
this should be configurable to allow at least to specify host and port based on https://istio.io/latest/docs/tasks/traffic-management/ingress/ingress-control/

Reply:
Likewise as previous comment!

Reply:
Taken Care!